resources/views/network/admin/internet_access.blade.php

```diff
@@ -134,13 +118,17 @@
                         minWidth: 150,
                     },
                     {
-                        title: "Internetelérés",
+                        title: "Netreg fizetett",
```

Comment:
The `false` and `true` values could be localised; but as they are only visible for admins anyway, it's not crucial.